src/main/java/com/revature/services/RSSService.java

```diff
@@ -0,0 +1,128 @@
+package com.revature.services;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import com.revature.DTOs.RSSAccountDTO;
+import com.revature.DTOs.RSSUserDTO;
+import com.revature.entities.User;
+import com.revature.repositories.UserRepository;
+
+@Service
+public class RSSService {
+	/*
```

Comment:
There is an @Author tag we should be putting above the methods

Reply:
Remove this comment block now that the @Author tags are being used correctly.